docs/teamshares/recipes/buttons.md

```diff
@@ -0,0 +1,190 @@
+# How to convert SharedUI::Button usages to Shoelace
+
+## 1. Link button
+
+> 💡 Hint: these are buttons marked with `external: true` or marked with `render_as_link: true`
+
+### Before
+
+```
+= render SharedUI::ButtonComponent.new(\
+  text: "Schedule prep session",
+  link: "https://www.calendly.com/mickmoylan",
+  icon: "calendar_icon",
+  external: true,
+  method: "get",
+  size: "medium",
+  type: "outlined")
+```
+
+### After
+
+```
+sl-button[size="large" pill href="https://www.calendly.com/mickmoylan" target="_blank"]
+  sl-icon[slot="prefix" name="calendar"]
+  | Schedule prep session
+```
+
+> 💡 Hint: use `target="_blank"` to open the link in a new tab
+
+## 2. Submit button
+
+> 💡 Hint: these are commonly used with simpleform and have `submit: true`
+
+### Before
+
+```
+= render SharedUI::ButtonComponent.new(\
+          text: "Accept",
+          type: "primary",
+          data:  { test_id: "accept-event-button" },
+          submit: true,
+          disabled: preview ? true : false,
+          options: { classes: "ml-2", name: "accept", value: "Accept" })
+```
+
+### After
+
+```jsx
+sl-button[size="large" pill type="submit" data-test_id="accept-event-button" variant="primary" disabled="#{preview ? true : false}"]
```

Comment:
Set the block format to `pug` instead of `jsx` and break the attributes to multiple lines